bindings/node.js/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Exists as a test harness for building and running tests in Linux
```

Comment:
whereas `make test` compiles and runs on whatever your local system is, `make linux-test` uses this specific Linux Docker environment.